airport-domain-model.md

```diff
@@ -0,0 +1,104 @@
+#### User Story 1
+
+As an air traffic controller
+So I can get passengers to a destination
+I want to instruct the airport to land a plane
+
+1. There is an airport noun(object).
+2. There is a plane noun(object).
+3. I am wanting to land(add) a plane at an airport.
+
+| Objects | Properties                   | Messages          | Outputs |
+| ------- | -------------------------    | ----------------- | ------- |
+| Airport | airportPlanes @Array[@Plane] | landPlane(@Plane) | @Void   |
+|         |                              |                   |         |
+| Plane   | id @String                   | getId()           | @String |
+
+### Tests
+
+1. Need an airport
+2. Need to be able to land plane at airport
+3. Airport planes length should increase by 1 when a plane has landed at airport
+
+**Test 1** - `airportPlanes` length increases when `landPlane` is called with *instance of Plane*
+
+**Test 2** - `landPlane` should only land 'Plane' instances to the 'airportManifest'
+
+**Test 3** - Edge Case - falsy values should not be added to Airport
+
+#### User Story 2 & 3
+
+As the system designer
+So that the software can be used for many different airports
+I would like a default airport capacity that can be overridden as appropriate
+
+As an air traffic controller
+To ensure safety
+I want to prevent landing when the airport is full
+
+1. Set a maximum airport capacity
+2. Override the maximum airport capacity to a different size limit using a method
+3. Check that more planes cannot be added beyond set airport capacity
+
+| Objects | Properties                            | Messages             | Outputs |
+| ------- | -------------------------             | -----------------    | ------- |
+| Airport | airportPlanes @Array[@Plane]          | landPlane(@Plane)    | @Void   |
+|         | MaximumCapacity @integer              | maximumCapacity()    | @Void   |
+|         |                                       |                      |         |
+| Plane   | id @String                            | getId()              | @String |
+
```

Comment:
I ended up combining these two user stories as while writing the code for setting the capacity, I inadvertently covered the 3rd User story criteria as well.